editions/tw5.com/tiddlers/system/wikitext-macros.tid

```diff
@@ -1,104 +1,96 @@
-open the ''example edition'' in a new window
+\procedure open-external-window(url)
+\whitespace notrim
+<$button actions=<<activateEditionWindow <<url>> class="tc-btn-invisible tc-tiddlylink">
+	open the ''example edition'' in a new window
 </$button>
 \end
 
 
-\define wikitext-example(src)
+\procedure wikitext-example(src)
 <div class="doc-example">
-
-<$macrocall $name="copy-to-clipboard-above-right" src=<<__src__>>/>
-
-<$codeblock code=<<__src__>>/>
-
-That renders as:
-
-<$macrocall $name="__src__"/>
-
-... and the underlying HTML is:
-
-<$wikify name="html" text=<<__src__>> output="html">
-<$codeblock code=<<html>>/>
-</$wikify>
+	<$macrocall $name="copy-to-clipboard-above-right" src=<<src>>/>
+	<$codeblock code=<<src>>/>
+	<p>
+		That renders as:
+	</p>
+	<$transclude $variable="src" $mode="block"/>
+	<p>
+		... and the underlying HTML is:
+	</p>
+	<$wikify name="html" text=<<src>> output="html">
+		<$codeblock code=<<html>>/>
+	</$wikify>
 </div>
 \end
 
-\define wikitext-example-without-html(src)
+\procedure wikitext-example-without-html(src)
 <div class="doc-example">
-
-<$macrocall $name="copy-to-clipboard-above-right" src=<<__src__>>/>
-
-<$codeblock code=<<__src__>>/>
-
-That renders as:
-
-<$macrocall $name="__src__"/>
-
+	<$macrocall $name="copy-to-clipboard-above-right" src=<<src>>/>
+	<$codeblock code=<<src>>/>
+	<p>
+		That renders as:
+	</p>
+	<$macrocall $name="src"/>
 </div>
 \end
 
-\define wikitext-example-table-header() <thead><tr><th/><th>wiki text</th><th>renders as</th></tr></thead>
-
-\define wikitext-example-table-row(id, code)
- <tr>
-<th><<__id__>></th><td><$codeblock code=<<__code__>>/></td><td>
+\procedure wikitext-example-table-header() <thead><tr><th/><th>wiki text</th><th>renders as</th></tr></thead>
 
-<<__code__>>
-</td>
+\procedure wikitext-example-table-row(id, code)
+<tr>
+	<th><<id>></th>
+	<td><$codeblock code=<<code>>/></td>
+	<td><<code>></td>
 </tr>
 \end
 
-\define tw-code(tiddler)
+\procedure tw-code(tiddler)
 <$codeblock language={{$tiddler$!!type}} code={{$tiddler$}}/>
 \end
 
-\define tw-code-link(tiddler)
+\procedure tw-code-link(tiddler)
 [[$tiddler$]]:
-
 <<tw-code $tiddler$>>
 \end
 
-\define flex-card(class,bordercolor:"",backgroundcolor:"",textcolor:"",imageField:"image",captionField:"caption",subtitle:"",descriptionField:"description",linkField:"link")
-\whitespace trim
-<$link class={{{ [<__class__>addprefix[tc-card ]] }}} to={{{ [<currentTiddler>get<__linkField__>else<currentTiddler>] }}}>
-	<div class="tc-card-accent" style.borderTop={{{ [<__bordercolor__>!is[blank]addprefix[5px solid ]] }}} style.background={{!!background}} style.backgroundColor=<<__backgroundcolor__>> style.color=<<__textcolor__>> style.fill=<<__textcolor__>>>
+\procedure flex-card(class,bordercolor:"",backgroundcolor:"",textcolor:"",imageField:"image",captionField:"caption",subtitle:"",descriptionField:"description",  inkField:"link")
```

Comment:
@Jermolene and @pmario
there is a wrong parameter name `inkField`, it should be `linkField`

Reply:
@BramChen -- thanks for the info. I did fix it with pr https://github.com/Jermolene/TiddlyWiki5/pull/8059. I did probably hit the delete button too often ;)